pom.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-jai</artifactId>
```

Comment:
For the artifactId, I thought we previously discussed making it unique so it wouldn't clash with other forks of the same jar?

For the version number, we can certainly keep it the same as bioformats, but if we change the artifactId, I thought it would make sense to restart at 1.0.0.

No strong opinion either way, but I thought there was some mention of clashing with other instances a while back, maybe in Fiji?